src/engraving/dom/score.cpp

```diff
@@ -4652,6 +4652,8 @@ ChordRest* Score::findChordRestEndingBeforeTickInTrack(const Fraction& tick, tra
 
 ChordRest* Score::cmdNextPrevSystem(ChordRest* cr, bool next)
 {
+    assert(cr);
+
```

Comment:
This would only work (well, abort) in Debug builds.
If you'd use 
```cpp
    IF_ASSERT(cr) {
        return;
    }
```
It'd sill abort there in Debug builds, but return in non-Debug buils (with logging the issue)